lib/typescript/websocketclient/index.ts

```diff
@@ -0,0 +1,91 @@
+import {StompWrapper} from './stompWrapper';
+import {Message, Channel} from 'httpclient';
```

Comment:
cannot we do all in the same import?

Reply:
the mappers are not part of the default export or are they?

Reply:
they should, I think they are

Reply:
but it's ok